proto/asn1-pdu/common.h

```diff
@@ -0,0 +1,68 @@
+// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////////
+
+#ifndef PROTO_ASN1_COMMON_H_
+#define PROTO_ASN1_COMMON_H_
+
+#include <math.h>
```

Comment:
It's not clear that you need this header here.

I suspect this is used in one or more `.cc` files, and if so, you should place the include directly in there.

I mentioned elsewhere on this CL, but the rule being "only include what you need" and "include everything you need" as the general rule of thumb :)